### ISSUE TITLE
Show how long left before stake withdrawing is allowed

### ISSUE THREAD

**Issue by ilblackdragon** [near/near-wallet#1387]
Currently after unstaking, it's unclear how long is left before one can withdraw funds from staking pool. Would be great to show some progress bar or approximate timer (36h left until withdrawal).

Also separately, given the clock reset when unstake again, it make sense to inform user about that if they have funds that are in progress of unstaking.

**Comment by jimmy3dita** [comment]
@corwinharrell happy to scope down this one, as I have some familiarity with the subject 😄

**Comment by corwinharrell** [comment]
yep happy to work on this with @Patrick1904. It should be pretty low-lift from a UI perspective, as long as we know what information is available to us to display (ie what calls we need to make to determine time elapsed etc. or if we just need to provide an approximate window).

**Comment by jimmy3dita** [comment]
Awesome, will find the right way to translate this into action
